Y/Yosys/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Yosys"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/YosysHQ/yosys.git", "2e421feb0ea526468493ab008f3c72beb12c2bc6")
+]
+
+dependencies = [
+    Dependency("boost_jll"; compat="=1.76.0"),
+    Dependency("Tcl_jll"),
+    Dependency("Readline_jll"),
+    Dependency("Libffi_jll"; compat="~3.2.2"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd yosys
+make CONFIG=${CC} -j${nproc}
```

Comment:
Umpf, of course that doesn't work
```suggestion
if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-freebsd* ]]; then
    CONFIG=clang
else
    CONFIG=gcc
fi
make CONFIG=${CONFIG} -j${nproc}
```